src/node/stage/approved.rs

```diff
@@ -1660,22 +1715,21 @@ impl Approved {
         // on our `OurKey` so they know our latest key. Need to vote for it first though, to
         // accumulate the signatures.
         if let Some(prefix) = sibling_prefix {
+            self.send_send_neighbour_info()?;
```

Comment:
Why are we doing this? We are not proactively updating neighbour sections about our section in any other case, so why here?

Reply:
same reason as mentioned above. mainly for the split case

Reply:
addressed by PR https://github.com/maidsafe/routing/pull/2170